src/admin/adminLayout.component.tsx

```diff
@@ -37,7 +32,9 @@ function AdminLayout() {
 
   const adminBreadCrumbs: BreadcrumbsInfo | undefined = adminPageName
     ? {
-        trail: [adminBreadCrumbsTrails[adminPageName] ?? ['', '']],
+        trail: adminBreadCrumbsTrails[adminPageName]
+          ? [adminBreadCrumbsTrails[adminPageName]]
+          : [],
```

Comment:
According to the types its never empty, always a string? So does it need the if or otherwise couldn't it be on one line like before potentially with `||` instead of `??`.

Reply:
I've refactored it to use the same type as the breadcrumbInfo